src/foolscap/banana.py

```diff
@@ -180,7 +180,7 @@ def initSlicer(self):
         assert tokens.IRootSlicer.providedBy(self.rootSlicer)
 
         itr = self.rootSlicer.slice()
-        next = iter(itr).next
+        next = iter(itr).__next__
         top = (self.rootSlicer, next, None)
```

Comment:
should probably use something besides `next` as the variable name here (shadows a builtin)

Reply:
Fixed global shadowing.